.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Maven PR Build
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    name: Maven PR Builder (JDK ${{ matrix.java }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 11 ]
```

Comment:
Is there a reason to not do 17 here too?

Reply:
Tried it on an earlier run but it failed: https://github.com/UCLALibrary/libcal-proxy/runs/6517168845?check_suite_focus=true#step:4:113

Reply:
And since we're deploying with Java 11 anyway I guess I don't understand why we should build with 17

Reply:
I'd like us to start moving projects towards 17 since that's the current LTS, but that shouldn't hold us up.

Reply:
I am in favor of using 17 for this project, fwiw